### DISCUSSION TITLE
JWMMF not showing / displaying PDF's.

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#261] [General]
Perhaps I am failing to use the software properly but as mentioned in another thread (thought it should have its own) I cannot get PDF's to display. 

Video HERE:

https://user-images.githubusercontent.com/35364657/164104533-45e058f8-0731-4e71-a945-c7544cf9e125.mp4

You mentioned 'dependencies' - keen to try any if you have any to share. 

Kind regards
Simon

**Comment by sircharlo** [comment]
Hi again!
Yes, for the current version, PDF and SVG conversion are unfortunately only triggered when MP4 conversion is enabled. This is a holdover from before hybrid and in-person meetings were the norm, and an oversight on my part.
As of next release, PDF and SVG files will always be converted, to allow them to be used with media presentation mode.

**Reply by Chipperchap** [reply]
Great!  Looking forward to it  🥇
